spec/datadog/tracing/contrib/concurrent_ruby/integration_test_spec.rb

```diff
@@ -135,6 +135,18 @@
           end
         end
       end
+
+      context 'when propagates without an active trace' do
+        it 'creates a root span' do
+          future = Concurrent::Promises.future do
+            tracer.trace('inner_span') {}
+          end
+
+          future.wait
+
+          expect(inner_span).to be_root_span
```

Comment:
I understand that it indirectly test that it did not crash (because if it crashed the test would blow up), but does that actually test that it didn't append a span to the trace?